src/server/Routes/Auth.js

```diff
@@ -1,44 +1,61 @@
 const express = require('express');
```

Comment:
We dont need any implementation just update the document.md files with the required routes.